PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -645,15 +664,74 @@ struct FlowGFWPbPb {
     FillProfile(corrconfigs.at(2), BootstrapArray[SampleIndex][kc26], centrality);
     FillProfile(corrconfigs.at(3), BootstrapArray[SampleIndex][kc28], centrality);
     FillProfile(corrconfigs.at(4), BootstrapArray[SampleIndex][kc22etagap], centrality);
-    FillProfile(corrconfigs.at(5), BootstrapArray[SampleIndex][kc32etagap], centrality);
-    FillProfile(corrconfigs.at(6), BootstrapArray[SampleIndex][kc34], centrality);
+    FillProfile(corrconfigs.at(5), BootstrapArray[SampleIndex][kc32], centrality);
+    FillProfile(corrconfigs.at(6), BootstrapArray[SampleIndex][kc32etagap], centrality);
+    FillProfile(corrconfigs.at(7), BootstrapArray[SampleIndex][kc34], centrality);
 
     // Filling Flow Container
     for (uint l_ind = 0; l_ind < corrconfigs.size(); l_ind++) {
       FillFC(corrconfigs.at(l_ind), centrality, l_Random);
     }
 
   } // End of process
+  PROCESS_SWITCH(FlowGFWPbPb, processData, "Process analysis for Run 3 data", false);
+
+  // Filter the Reco tracks
+  Filter mytrackFilter = (nabs(aod::track::eta) < cfgCutEta) && (aod::track::pt > cfgCutPtMin) && (aod::track::pt < cfgCutPtMax) && (nabs(aod::track::dcaXY) < cfgCutDCAxy);
+  using myTracks = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra, aod::TrackSelection, aod::TracksDCA, aod::McTrackLabels>>;
+  using myCollision = soa::Join<aod::Collisions, aod::CentFT0Cs>;
```

Comment:
Here `myCollision` is a table so keep the plural, if not the name is misleading
See also the below comment

Reply:
You are right, I will change it to `myCollisions ` Thanks!